backends/advanced/tests/test_integration.py

```
@@ -24,13 +24,15 @@
 - MongoDB on port 27018 (vs dev 27017)
 - Qdrant on ports 6335/6336 (vs dev 6333/6334)
 - Parakeet ASR on port 8767 (parakeet provider)
-- Pre-configured test credentials in .env.test
+- Test credentials configured via environment variables
 - Provider selection via TRANSCRIPTION_PROVIDER environment variable
 """
 
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)